packages/ui-components/package.json

```diff
@@ -16,14 +16,12 @@
     "test": "echo skipped"
   },
   "dependencies": {
-    "@polkadot/types": "^0.48.1",
-    "@polkadot/ui-identicon": "^0.31.1",
-    "@polkadot/ui-keyring": "^0.31.1",
-    "@polkadot/util": "^0.38.1",
+    "@polkadot/ui-identicon": "^0.32.1",
+    "@polkadot/ui-keyring": "^0.32.1",
     "@types/styled-components": "4.1.0",
     "react-dropzone": "^10.0.0",
     "semantic-ui-react": "^0.86.0",
-    "styled-components": "4.1.0"
+    "styled-components": "^4.1.3"
```

Comment:
styled components version $^4.1.* was causing a series of hard to debug errors before, which is why it is currently hardcoded to 4.1.0 while the DefinitielyTyped packages is being fixed. I had no issues building this time, but I'm still hesitant to upgrade the patch till we're 100% safe there, unless the patch is stricly necessary for something else.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33015

Reply:
https://github.com/paritytech/substrate-light-ui/issues/70

Reply:
As written in chat: ui-keyring needs ^4.1.3, and if I put 4.1.0 as before, then there are 2 instances of styled components, which makes some styles crash

Reply:
Umpf. Yes that was added recently since @Stefie pulled it into an external project and detected it missing. (Generally try to keep to peerDeps there since it can create clashes in downstream) These deps are a screw-up.

Reply:
Ah, yeah, for me it was actually the `ui-identicon` module that I couldn't use because  `styled-components` was missing as a dependency. But I also added it to the ui-keyring package because I saw that it was missing there, too